src/spec-common/dotfiles.ts

```diff
@@ -44,20 +44,34 @@ export async function installDotfiles(params: ResolverParameters, properties: Co
 ${createFileCommand(markerFile)} || (echo dotfiles marker found && exit 1) || exit 0
 command -v git >/dev/null 2>&1 || (echo git not found && exit 1) || exit 0
 [ -e ${targetPath} ] || ${allEnv}git clone ${repository} ${targetPath} || exit $?
-if [ -x ${installCommand} ]
+echo Setting current directory to ${targetPath}
+cd ${targetPath}
+
+if [ -f ${targetPath}/${installCommand} ]
 then
-	echo Executing script ${installCommand}
-	cd ${targetPath} && ${allEnv}${installCommand}
+	if [ ! -x ${targetPath}/${installCommand} ]
```

Comment:
We should quote `"${targetPath}/${installCommand}"` (and other variable usages), so things don't break when there are spaces in the path or filename.